input/fsh/omrs-allergyIntolerance.fsh

```diff
@@ -5,19 +5,16 @@ Title:    "OpenMRS AllergyIntolerance"
 Description: "A FHIR Allergy as understood by OpenMRS"
 * identifier 0..1
 * code 0..1
-* category 0..*
 * reaction 0..1
 * type  0..1
 * clinicalStatus 0..1
 * verificationStatus 0..1
+* patient 1..1
+* recorder 0..1
+* type 0..1
 * criticality 0..1
-* encounter 0..1
```

Comment:
Hi, Sharif don't completely remove the attributes which are not supported , provide 0..0 cardinality